Reduced scope of Shell section

julianpistorius opened the pull request.

julianpistorius committed:
Reduced scope of Shell section

uhilgert commented:
Hi Julian, would you please share the content of this comment in real text? I have no idea what it is supposed to mean nd github is not an easy one to give its secrets away... Thanks, Uwe

---

From: Julian Pistorius notifications@github.com
Sent: Thursday, June 2, 2016 3:51 PM
To: uhilgert/2016-06-11-Tucson
Subject: [uhilgert/2016-06-11-Tucson] Reduced scope of Shell section (#1)

---

You can view, comment on, or merge this pull request online at:

  https://github.com/uhilgert/2016-06-11-Tucson/pull/1

Commit Summary
-   Reduced scope of Shell section

File Changes
-   M index.htmlhttps://github.com/uhilgert/2016-06-11-Tucson/pull/1/files#diff-0 (3)

Patch Links:
-   https://github.com/uhilgert/2016-06-11-Tucson/pull/1.patch
-   https://github.com/uhilgert/2016-06-11-Tucson/pull/1.diff

## 

You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHubhttps://github.com/uhilgert/2016-06-11-Tucson/pull/1, or mute the threadhttps://github.com/notifications/unsubscribe/AE6PsyqyWJozG0egPBZS_bUuxi6EpfLcks5qH15ngaJpZM4ItDpD.

julianpistorius commented:
Hi Uwe. If you click on the "Files changed" link at the top of this page you'll see the changes. But in brief:
1. I removed "Looping over files" and "Creating and running shell scripts".
2. I added "Introducing shell"

uhilgert commented:
Ok, thanks.

---

From: Julian Pistorius notifications@github.com
Sent: Thursday, June 2, 2016 8:43 PM
To: uhilgert/2016-06-11-Tucson
Cc: Hilgert, Uwe K.K - (hilgert); Comment
Subject: Re: [uhilgert/2016-06-11-Tucson] Reduced scope of Shell section (#1)

Hi Uwe. If you click on the "Files changed" link at the top of this page you'll see the changes. But in brief:
1.  I removed "Looping over files" and "Creating and running shell scripts".
2.  I added "Introducing shell"

## 

You are receiving this because you commented.
Reply to this email directly, view it on GitHubhttps://github.com/uhilgert/2016-06-11-Tucson/pull/1#issuecomment-223483902, or mute the threadhttps://github.com/notifications/unsubscribe/AE6Ps9egHHKKplNzpiNiteTZiIsk4X1Tks5qH6LugaJpZM4ItDpD.

uhilgert commented:
How did the meeting go? Where detailed plans for each module developed? Was Andre there? I would like to understand whether each instructor understands what data to use? Thanks.

---

From: Julian Pistorius notifications@github.com
Sent: Thursday, June 2, 2016 8:43 PM
To: uhilgert/2016-06-11-Tucson
Cc: Hilgert, Uwe K.K - (hilgert); Comment
Subject: Re: [uhilgert/2016-06-11-Tucson] Reduced scope of Shell section (#1)

Hi Uwe. If you click on the "Files changed" link at the top of this page you'll see the changes. But in brief:
1.  I removed "Looping over files" and "Creating and running shell scripts".
2.  I added "Introducing shell"

## 

You are receiving this because you commented.
Reply to this email directly, view it on GitHubhttps://github.com/uhilgert/2016-06-11-Tucson/pull/1#issuecomment-223483902, or mute the threadhttps://github.com/notifications/unsubscribe/AE6Ps9egHHKKplNzpiNiteTZiIsk4X1Tks5qH6LugaJpZM4ItDpD.
